gctscript/vm/gctscript.go

```diff
@@ -0,0 +1,82 @@
+package vm
+
+import (
+	"fmt"
+
+	"github.com/gofrs/uuid"
+	"github.com/thrasher-corp/gocryptotrader/gctscript/wrappers/validator"
+	log "github.com/thrasher-corp/gocryptotrader/logger"
+)
+
+// New returns a new instance of VM
+func New() *VM {
+	if AllVMs == nil {
+		AllVMs = make(map[uuid.UUID]*VM)
+	}
+
+	if len(AllVMs) >= int(GCTScriptConfig.MaxVirtualMachines) {
+		log.Warnln(log.GCTScriptMgr, "GCTScript MaxVirtualMachines hit unable to start further instances")
```

Comment:
Grammar change: GCTScript MaxVirtualMachines hit, unable to start further instances